### ISSUE TITLE
Facebook crashed

### ISSUE THREAD

**Issue by akha666** [0bbedCode/XPL-EX#12]
Facebook app crashs, but opens after 3 times

**Comment by ThreeDeeJay** [comment]
XPL on Facebook sounds like pouring holy water on a demon. It's gonna squeal and put up a fight before going down lol
Anyway, you should narrow down the hook(s) that's crashing and the error log (a ! sign to the left of unstable hooks) so the dev has something to work with.
Or just uninstall Facebook, permanently 🙃

**Comment by 0bbedCode** [comment]
Agree do you have any logs, what hooks did you select what hook did you de-select anything more informative ?

**Comment by akha666** [comment]
First I got the and en exclamation mark "!" beside the "Spoof User Agent". furthermore, I have tried to deselect everything except the "Stop Automatically". But Facebook app keeps crash and some time loses the app setting and dark mode.
How can I export the log for sharing?

**Comment by 0bbedCode** [comment]
yes that "!" is the logs click it don't be scared ! use the fingers god gifted you !! Send me that message and or a screenshot

**Comment by 0bbedCode** [comment]
To be clear when you have (0) Hooks selected (0) Groups selected it still crashes ??? You made clear you only have the option selected "Stop Automatically" and if you uncheck that ?
